nx-workspace/apps/web/src/components/atoms/core/index.tsx

```diff
@@ -1,34 +1,28 @@
-export { Alert, AlertProps, Severity } from "./Alert";
-export { AppBar, AppBarProps } from "./AppBar";
-export { Box, BoxProps } from "./Box";
-export { Button, ButtonProps } from "./Button";
+export * from "./Alert";
+export * from "./AppBar";
+export * from "./Box";
+export * from "./Button";
 export { ClearIcon } from "./ClearIcon";
```

Comment:
Hmm, I thought being explicit here might be better, but if we go with `*`, maybe use it for every export?